binding/Binding/SKBitmap.cs

```diff
@@ -401,6 +456,14 @@ public void SetColorTable (SKColorTable ct)
 
 		// more properties
 
+		public SKPixmap Pixmap {
+			get {
+				var pixmap = new SKPixmap (SkiaApi.sk_bitmap_get_pixmap (Handle), false); // pixmap is owned by this
+				pixmap.pixelSource = this;
+				return pixmap;
+			}
+		}
```

Comment:
What happens if there is no pixels? Will the handle be null? Can you write a test for the cases where there are no pixels?

Reply:
if there are no pixels then i assume pixmap will be empty